Packs/EmailCommunication/Scripts/DisplayEmailHtmlThread/DisplayEmailHtmlThread.py

```diff
@@ -55,41 +55,60 @@ def html_cleanup(full_thread_html):
     return final_html_result
 
 
+def remove_style_and_color(html_message):
+    demisto.debug(f'DisplayEmailHtmlThread - {html_message=}')
+    parsed_html_body = BeautifulSoup(html_message, 'html.parser')
+
+    # Remove all style tags
+    for style_tag in parsed_html_body.find_all('style'):
+        style_tag.decompose()
+
+    # Remove all style attributes from other tags
+    for tag in parsed_html_body.find_all(True):
+        if 'style' in tag.attrs:
+            del tag.attrs['style']
+        if 'color' in tag.attrs:
+            del tag.attrs['color']
+    demisto.debug(f'DisplayEmailHtmlThread - after removing the style and color from the html {str(parsed_html_body)=}')
+    return str(parsed_html_body)
+
+
 def main():
     incident = demisto.incident()
     custom_fields = incident.get('CustomFields')
     thread_number = custom_fields.get('emailselectedthread', 0)
     incident_context = demisto.context()
     email_threads = incident_context.get('EmailThreads', {})
+    demisto.debug(f'DisplayEmailHtmlThread - {email_threads=}')
```

Comment:
Will this log be more than 2 lines? If so, maybe it is with logging something else in order not to trash the logs.